data_fusion/interface_modules/output_formats/layered_imagery.py

```diff
@@ -20,8 +20,13 @@
 from geoips.image_utils.mpl_utils import save_image, plot_overlays
 from geoips.image_utils.mpl_utils import set_title, create_colorbar
 from geoips.interface_modules.procflows.single_source import plot_data
+
+# New geoips interface classes
+from geoips.interfaces import colormaps
+# from geoips.dev.cmap import get_cmap
```

Comment:
Should this just be removed? It seems like it will just become cruft.

Reply:
Removed